docs/bot/setup_debian.md

```diff
@@ -0,0 +1,138 @@
+# Setting up Draupnir on Debian
```

Comment:
```suggestion
# Installation on Debian
```

Probably needs to be like this to be consistent with the other title "Installation with docker and systemd".